package.json

```diff
@@ -83,6 +83,7 @@
     "pretty-fast": "^0.2.3",
     "prop-types": "^15.6.0",
     "react": "16.2.0",
+    "react-collapsible": "^2.0.4",
```

Comment:
I don't see this being used anywhere; is this an artifact of experimentation?  If so, please remove this and the `package-lock.json` change.